guides/source/upgrading_ruby_on_rails.md

```diff
@@ -579,10 +744,10 @@ end
 
 ### Digest class for ActiveSupport::Digest changing to SHA256
 
-The default digest class for ActiveSupport::Digest is changing from SHA1 to SHA256.
-This has consequences for things like Etags that will change and cache keys as well.
-Changing these keys can have impact on cache hit rates, so be careful and watch out
-for this when upgrading to the new hash.
+The default digest class for ActiveSupport::Digest is changing from SHA1 to
+SHA256. This has consequences for things like Etags that will change and cache
+keys as well. Changing these keys can have impact on cache hit rates, so be
+careful and watch out for this when upgrading to the new hash.
```

Comment:
```suggestion
The default digest class for `ActiveSupport::Digest` is changing from SHA1 to 
SHA256. This affects values such as ETags and cache keys, which will also 
change. Since altering these keys can impact cache hit rates, monitor your 
application when upgrading to the new hash.
```